### PR TITLE
Transform package names for case insensitive ecosystems

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

**Comment by fridex** [comment]
Related: https://github.com/fabric8-analytics/fabric8-analytics-worker/pull/23

File: bayesian/api_v1.py [hunk 1]
```diff
@@ -248,6 +248,7 @@ class ComponentAnalyses(ResourceWithSchema):
     def get(self, ecosystem, package, version):
         if ecosystem == 'maven':
```
Comment (fridex) [inline bayesian/api_v1.py]: Yes, that is what I suggested in https://github.com/fabric8-analytics/fabric8-analytics-worker/pull/23#issuecomment-303336495

**Comment by fridex** [comment]
[test this]

**Comment by fridex** [comment]
Once https://github.com/fabric8-analytics/fabric8-analytics-server/pull/17 gets merged, CI will be green.

**Comment by jpopelka** [comment]
[test this]

**Comment by msrb** [comment]
LGTM, thanks 😉

**Pull request merged**